addons/website/models/website_visitor.py

```diff
@@ -209,6 +209,11 @@ def _upsert_visitor(self, access_token, force_track_values=None):
         country_id = request.env['res.country'].sudo().search([
             ('code', '=', country_code)
         ], limit=1).id if country_code else None
```

Comment:
```suggestion
        ], limit=1).id or None if country_code else None
```

Reply:
That would be the obvious fix (or below: `'country_id': country_id,` -> `'country_id': country_id or None,`
But I'd like to keep as is, encapsulated in its own condition with an easy to understand comment, given the weirdness of the fix. It's yet unsure if someone messed with this GEOIP db or his Odoo countries.